builder/pwnagotchi.yml

```diff
@@ -224,7 +230,7 @@
 
   - name: clone pwnagotchi repository
     git:
-      repo: https://github.com/evilsocket/pwnagotchi.git
+      repo: https://github.com/scifijunk/pwnagotchi.git
```

Comment:
You forgot to change this back

Reply:
At the time I made the pr it was changed back. I then changed back to mine to play with it further. Now since I now know it is being looked at I change it back so it will fit.